src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -12764,10 +12764,7 @@ Unit* Creature::SelectVictim()
         return NULL;
 
     if (target && _IsTargetAcceptable(target) && CanCreatureAttack(target))
-    {
-        SetInFront(target);
         return target;
```

Comment:
Good question, will test this asap.. Shouldn't SPELL_ATTR0_CAST_TRACK_TARGET take care of this ?